scripts/build-slugs-table.js

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+/**
+ * @fileoverview
+ * Generates a MarkDown file that lists every brand name and their slug.
+ */
+
+const fs = require("fs");
+const path = require("path");
+
+const dataFile = path.resolve(__dirname, "..", "_data", "simple-icons.json");
+const slugsFile = path.resolve(__dirname, "..", "slugs.md");
+
+const data = require(dataFile);
+const { titleToSlug } = require("./utils.js");
+
+let content = `<!--
+This file is automatically generated. If you want to change something, please
+update the script at '${__filename.replace(__dirname, "scripts")}'.
+-->
+
+# Simple Icons slugs
+
+| Brand name | Brand slug |
+| :--- | :--- |
+`;
+
+data.icons.forEach(icon => {
+  const brandName = icon.title;
+  const brandSlug = titleToSlug(icon.title);
+  content += `| \`${brandName}\` | \`${brandSlug}\` |\n`
```

Comment:
Should we also display the icon there?
```
content += `| <img src="./icons/${brandSlug}.svg" alt="${brandName}" width=24 height=24> | \`${brandName}\` | \`${brandSlug}\` |\n`
```

Reply:
Interesting idea! Not sure if it is useful but if more people are in favor of this I don't mind adding it :slightly_smiling_face:

That is, provided that it doesn't make the table (significantly) bigger or more ugly.

Reply:
Probably one to come back to after GitHub come up with a solution for displaying different images for different themes.